embedchain/vectordb/zilliz.py

```diff
@@ -222,3 +222,16 @@ def set_collection_name(self, name: str):
         if not isinstance(name, str):
             raise TypeError("Collection name must be a string")
         self.config.collection_name = name
+
+    def delete(self, keys: Union[list, str, int]):
```

Comment:
Embedchain user would not know what the primary keys are for the entries. I think we need to give the delete option by the doc_id which is exposed to the user.

Reply:
Zilliz does not allow deleting by querying. It only allows deleting with pks which is `id` in our case.

[Ref doc](https://docs.zilliz.com/reference/delete-1)